lib/web_ui/lib/src/engine/keyboard.dart

```diff
@@ -50,30 +50,57 @@ class Keyboard {
   static const JSONMessageCodec _messageCodec = JSONMessageCodec();
 
   void _handleHtmlEvent(html.KeyboardEvent event) {
+    if (event.repeat) {
+      // We don't care about repeat events. All we need is keydown and keyup.
```

Comment:
Interesting. Would this allow us to respect the OS-level key repeat frequency configuration?

Reply:
I changed the code to send repeat events to Flutter. It seems to be working fine at the moment. If it becomes a problem in the future, we can revisit (cc @gspencergoog).